erpnext/hr/doctype/leave_application/leave_application.py

```diff
@@ -125,7 +125,7 @@ def update_attendance(self):
 				status = "Half Day" if date == self.half_day_date else "On Leave"
 
 				attendance_name = frappe.db.exists('Attendance', dict(employee = self.employee,
```

Comment:
@vijaywm Use frappe.db.get_value instead of frappe.db.exists since it just returns `True` or `False` and not the name of the record. `attendance_name` is used below to get the `Attendance` Doc and update it. Test case is also failing because of it. Please make the required changes

Reply:
![image](https://user-images.githubusercontent.com/4676890/66801974-947df300-ef38-11e9-94b4-42735a31a844.png)
exists is returning None if not found else name of document  or list of document names
https://github.com/frappe/frappe/blob/df78b3647870261da86ad95bdf8a155204cf2e4d/frappe/database/database.py#L777

I checked Travis had failed because of some item_group test related error

Reply:
@vijaywm Following test case is also failing. Can you please check and fix that as well
![image](https://user-images.githubusercontent.com/42651287/67190458-9f8bc400-f40d-11e9-9812-cd472c1ad5e3.png)